src/sv-pipeline/04_variant_resolution/scripts/clean_vcf_part1.sh

```diff
@@ -27,18 +27,27 @@ zcat $vcf \
   >whitelist.txt
 
 ##convert EV integer back into string##
-zcat $vcf \
-      | awk '{print $0 "\t"}' \
-      | sed -e 's/:7'"\t"'/:RD,PE,SR'"\t"'/g' \
-      | sed -e 's/:6'"\t"'/:PE,SR'"\t"'/g' \
-      | sed -e 's/:5'"\t"'/:RD,SR'"\t"'/g' \
-      | sed -e 's/:3'"\t"'/:RD,PE'"\t"'/g' \
-      | sed -e 's/:2'"\t"'/:PE'"\t"'/g' \
-      -e 's/:4'"\t"'/:SR'"\t"'/g' \
-      -e 's/:1'"\t"'/:RD'"\t"'/g' \
-      |sed 's/'"\t"'$//g' \
-      |sed 's/ID=EV,Number=1,Type=Integer/ID=EV,Number=1,Type=String/g' \
-      | bgzip > EV.update.vcf.gz
+bcftools query -f "%CHROM\t%POS\t%REF\t%ALT\t[%EV\t]\n" $vcf > ev.tab
+cut -f1-4 ev.tab > ev.sites.tab
+cut -f5- ev.tab \
+  | sed -e 's/7/RD,PE,SR/g' -e 's/6/PE,SR/g' -e 's/5/RD,SR/g' -e 's/3/RD,PE/g' -e 's/2/PE/g' -e 's/4/SR/g' -e 's/1/RD/g' \
+  > ev.replaced.list
+paste ev.sites.tab ev.replaced.list | bgzip -c > ev.replaced.tab.gz
+tabix -s1 -b2 -e2 ev.replaced.tab.gz
+
+echo '##FORMAT=<ID=EV,Number=1,Type=String,Description="Classes of evidence supporting final genotype">' > ev_header.txt
+tabix $vcf
+bcftools annotate -x FORMAT/EV $vcf | bgzip -c > no_ev.vcf.gz
+rm $vcf $vcf.tbi
+
+bcftools annotate \
+  -a ev.replaced.tab.gz \
+  -c CHROM,POS,REF,ALT,FORMAT/EV \
+  -h ev_header.txt \
+  no_ev.vcf.gz \
+  | bgzip -c > EV.update.vcf.gz
+
+rm ev.tab no_ev.vcf.gz ev.replaced.tab.gz ev.replaced.tab.gz.tbi ev.replaced.list
```

Comment:
Is there a reason to remove these files? Just curious on whether you ran into some issue based on leaving them around.

Reply:
No just cleanup. I expect this could be worthwhile for gnomad.